lib/messages/pull-request.js

```diff
@@ -48,6 +59,10 @@ class PullRequest extends AbstractIssue {
         title: 'Comments',
         value: this.pullRequest.comments,
       },
+      {
+        title: 'Reviewers',
+        value: Array.from(reviewers).join(', '),
```

Comment:
Are there emojis that we can put here that map well to the: commented, requested, approved, requested_changes states? It's a long shot but maybe possible?

Reply:
That's a great idea! As we discussed offline I think we can do that in another issue because a user can do multiple reviews and we only want not the last one but the last one that is meaningful, because the last one can be a comment and there can be a previous one that is approved or request_changes,... so I think we can do that in another issue and keep this one simple. But I'm 100% for implementing it!